content/en/blog/2023/php-auto-instrumentation/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Opentelemetry PHP Auto-Instrumentation
+linkTitle: PHP Auto-Instrumentation
+date: 2023-03-15
+author: '[Przemek Delewski](https://github.com/pdelewski/) (Sumo Logic)'
+---
+
+Automatic Instrumentation is a process of adding tracing capabilities into user
+application without modyfing it's source code. There are several techniques to
+do that, but all of them more or less work in the same way by injecting
+additional code into original one during compile time, link time, run-time or by
+extending the operating system in case of [ebpf](https://ebpf.io/). This
+blogpost presents method used by Opentelemetry PHP auto-instrumentation.
+
+## Preconditions
+
+Due to the technique chosen by the opentelemetry-php community for
+auto-instrumentation purposes, there are a few preconditions that must be
+fulfilled before user can use it in their application. PHP auto-instrumentation
+is based on observability API that was introduced in PHP 8.0. This means that
+user has to have at least PHP 8 installed. Another important component is the
+[composer](https://getcomposer.org/download/) package manager in most recent
+version. Third, is availability of c compiler in order to build
+[c extension](https://github.com/open-telemetry/opentelemetry-php-instrumentation)
+which is core component used for injecting tracing code.
```

Comment:
```suggestion
## Prerequisites

To use the [PHP auto-instrumentation](https://github.com/open-telemetry/opentelemetry-php-instrumentation), you'll need three things:

- PHP 8.0 or higher. The PH auto-instrumentation used the Observability
API introduced in PHP 8.0.
- [Composer]((https://getcomposer.org/download/)
- A C Compiler must be available on your machine
```

I felt like this part needed a rewrite. Ideally we can take content from this blog and turn it into docs when things stabilize a little more. Shaping it a little more like docs for instructional stuff (not commentary) helps with that a bunch.